hw20/index.js

```diff
@@ -0,0 +1,110 @@
+'use strict'
+
+//TASK 1
+
+// const colors = ['red', 'green', 'blue'];
+
+// console.log(colors.length);
+
+//TASK 2
+
+// const animals = ['monkey', 'dog', 'cat'];
+
+// console.log(animals[animals.length -1]);
+
+//TASK 3
+
+// const numbers = [5, 43, 63, 23, 90];
+
+// numbers.shift();
+// numbers.shift();
+// numbers.shift();
+// numbers.shift();
+// numbers.shift();
+
+// numbers.unshift(90);
+// numbers.unshift(23);
+// numbers.unshift(63);
+// numbers.unshift(43);
+// numbers.unshift(5);
```

Comment:
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/unshift
```suggestion
// numbers.unshift(90, 23, 63, 43, 5);
```